README.md

```diff
@@ -1,8 +1,62 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C++, C#, C, JavaScript (Vue and ReactJS), MongoDB
+
+**Familiar:** .NET, Swift, Kotlin, Assembly language, Waterfall Methodology
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent to manage and insert resource usage information into a PostgreSQL database. The host usage information was collected through bash commands and automated using crontab.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/core_java)]:
+      
+  - Twitter App: In-progress
+  - JDBC App: Developed an application that uses Java Database Connectivity (JDBC) to insert and perform CRUD operations on data stored in a Docker-hosted PostgreSQL database. This project uses JDBC, PSQL, Maven, Docker, DAO, and DTOs.
+  - Grep App: Implemented an application to simulate the grep command available within Linux operating systems. This app was developed using Java, Maven, and Docker. The grep app allows a user to specify a regex, root directory, and output file. The files found within the root directory and any subsequent subdirectories are read then the retrieved lines are compared to the defined regex pattern. If the line matches the regex, it is then written to the output file.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**Electronic Medical Records System**: Created a proof-of-concept EMR system for Otekha Health. This project was built using ReactJS, Java RESTful server, Docker, MongoDB, MySQL, Redis, and HAPI-FHIR API.
```

Comment:
Created a proof-of-concept EMR system for Otekha Health using ReactJS, Java RESTful server, Docker, MongoDB, MySQL, Redis, and HAPI-FHIR API. Features, algo, desgin, achivement, results